functions/lib/extract.js

```diff
@@ -25,7 +25,8 @@ const getPayload = (snapshot) => {
         const [field, value] = processors_1.valueProcessor(firebaseField, snapshot.get(firebaseField));
         logs.debug('field', field);
         logs.debug('value', value);
-        if (value) {
+        // for fields with boolean data type, the value of false is a valid value.
+        if (value || value == false) {
```

Comment:
```suggestion
        // for fields with boolean data type, the value of false is a valid value.
        if (typeof value !== 'undefined') {
```

does this also test the same?